ConsentViewController/Classes/SourcePointClient/SourcepointClientCoordinator.swift

```diff
@@ -413,7 +413,7 @@ class SourcepointClientCoordinator: SPClientCoordinator {
             state.ccpa?.uuid = ccpa.uuid
             state.ccpa?.dateCreated = ccpa.dateCreated
             state.ccpa?.status = ccpa.status
-            state.ccpa?.uspstring = ccpa.uspstring
+//            state.ccpa?.uspstring = ccpa.uspstring
```

Comment:
pls remove `//` or delete the commented line

Reply:
Done